readmes/README.Linux.md

```diff
@@ -170,4 +174,56 @@ After you have installed all the dependencies for your operating system, it is t
     ssh <hostname of your machine>  # e.g., ssh briarwood (You can use `hostname` to get the hostname of your machine.)
     ```
 
-4. Set up your system configuration by following the installation guide on [README.md](https://github.com/cloudberrydb/cloudberrydb/tree/main/readmes).
+## Step 4. Build Cloudberry Database
+
+After you have installed all the dependencies and performed the prerequisite platform tasks, you can start to build Cloudberry Database. Run the following commands in sequence.
+
+```bash
+# Configure build environment to install at /usr/local/cloudberrydb
+./configure --with-perl --with-python --with-libxml --with-gssapi --prefix=/usr/local/cloudberrydb
+
+# Compile and install
+make -j8
+make -j8 install
+
+# Bring in greenplum environment for CBDB into your running shell
+source /usr/local/cloudberrydb/greenplum_path.sh
+
+# Start demo cluster
+make create-demo-cluster
+# (gpdemo-env.sh contains __PGPORT__ and __MASTER_DATA_DIRECTORY__ values)
+source gpAux/gpdemo/gpdemo-env.sh
+```
+
+- The directory and the TCP ports for the demo cluster can be changed on the fly.
+Instead of `make cluster`, consider:
+
+    ```bash
+    DATADIRS=/tmp/cbdb-cluster PORT_BASE=5555 make cluster
+    ```
```

Comment:
we don't have a task called `cluster`.

Reply:
Deleted in https://github.com/cloudberrydb/cloudberrydb/pull/224/commits/e4f98d846b0dcb58441f4c9d7dca8364f32df21d